test/uk/gov/hmrc/alcoholdutyaccount/models/subscription/AdrSubscriptionSummarySpec.scala

```diff
@@ -51,18 +72,46 @@ class AdrSubscriptionSummarySpec extends SpecBase {
     Seq(
       (hods.Beer, Set(AlcoholRegime.Beer)),
       (hods.CiderOrPerry, Set(AlcoholRegime.Cider, AlcoholRegime.OtherFermentedProduct)),
-      (hods.WineAndOtherFermentedProduct, Set(AlcoholRegime.Wine, AlcoholRegime.OtherFermentedProduct)),
-      (hods.Spirits, Set(AlcoholRegime.Spirits))
+      (hods.Wine, Set(AlcoholRegime.Wine, AlcoholRegime.OtherFermentedProduct)),
+      (hods.Spirits, Set(AlcoholRegime.Spirits)),
+      (hods.OtherFermentedProduct, Set(AlcoholRegime.OtherFermentedProduct))
+    ).foreach { case (approvedSubscription, expectedRegimes) =>
+      s"should return an AdrSubscriptionSummary of ${expectedRegimes.map(_.entryName).mkString(",")} when ${approvedSubscription.getClass.getTypeName} is approved and OFP subscription is not available" in {
+        val subscriptionSummary: SubscriptionSummary = SubscriptionSummary(
+          typeOfAlcoholApprovedFor = Set(approvedSubscription),
+          smallciderFlag = false,
+          approvalStatus = hods.Approved,
+          insolvencyFlag = false
+        )
+
+        val adrSubscriptionSummary = AdrSubscriptionSummary
+          .fromSubscriptionSummary(subscriptionSummary, ofpSubscriptionAvailable = false)
+          .toOption
```

Comment:
Is there a reason to use .toOption.get here rather than wrapping expectedRegimes in a right? I would think that if future iterations to this code went wrong, the issue would be easier to debug without a None.get exception

Reply:
yes that's cleaner

Reply:
Going to refactor these tests a bit.